var/spack/repos/builtin/packages/xterm/package.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Xterm(AutotoolsPackage):
+    """The xterm program is a terminal emulator for the X Window System. It
+    provides DEC VT102 and Tektronix 4014 compatible terminals for programs
+    that can't use the window system directly."""
+
+    homepage = "http://invisible-island.net/xterm/"
+    url      = "ftp://invisible-island.net/xterm/xterm.tar.gz"
+
```

Comment:
Apparently I didn't look close enough at the ftp repo. There is a rolling release, but I think it's just a symlink to the versioned tarballs. Fixed now!